### PR TITLE
Hummus Exchange Volume

### PR DESCRIPTION
**Pull request by devdevdevdevde**
website https://app.hummus.exchange/dashboard & defillama https://defillama.com/protocol/hummus-exchange

### CODE REVIEW

**Comment by llamatester** [comment]
The hummus adapter exports: 
        
 
 _______________________________________
    Dexs for 12/6/2023
    _______________________________________
    
    8095.210595000002
    METIS 👇
    Backfill start time: 30/8/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 8095.210595000002
    Timestamp: 1686528000
    
    
    
    
    Done in 6.04s.

File: dexs/hummus/index.ts [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      vol = args.fromAmount / 10 ** decimals[tokenIndex];
+    } else {
+      tokenIndex = tokens.findIndex((address) => address === args.toToken);
+      vol = args.toAmount / 10 ** decimals[tokenIndex];
+    }
+
+    dailyVolume += vol;
+  });
+
+  console.log(dailyVolume);
```
Comment (dtmkeng) [inline dexs/hummus/index.ts]: could you delete console log ?

**Review by dtmkeng — CHANGES_REQUESTED** [review]

File: dexs/hummus/index.ts (hunk 1 above)
Comment (devdevdevdevde) [reply dexs/hummus/index.ts]: yes

**Comment by devdevdevdevde** [comment]
Everything?

**Comment by llamatester** [comment]
The hummus adapter exports: 
        
 
 _______________________________________
    Dexs for 12/6/2023
    _______________________________________
    
    METIS 👇
    Backfill start time: 30/8/2022
    NO METHODOLOGY SPECIFIED
    Daily volume: 8095.210595000002
    Timestamp: 1686528000
    
    
    
    
    Done in 6.90s.

**Pull request merged**